example/elm.js

```diff
@@ -5404,16 +5404,20 @@ var _Bogdanp$elm_ast$Ast_Helpers$reservedOperators = {
 		}
 	}
```

Comment:
This file shouldn't be here

Reply:
`example/elm.js` have to be in the repo because of http://bogdanp.github.io/elm-ast/example/

Reply:
@Baransu it's only updated on branch `gitpages`